### PR TITLE
added detailed player status section

### PR DESCRIPTION
**Pull request by Arjun99ab**
renders below guild info
image

### CODE REVIEW

**Comment by amosyu2000** [comment]
Hi @Arjun99ab , thanks for taking the time to do this! I'll review soon.

File: src/constants/hypixel/Hypixel.js [hunk 1]
```diff
@@ -260,4 +260,221 @@ export const HYPIXEL = {
 		'stats.SkyClash.wins', // SkyClash (removed)
 		'stats.WoolGames.wool_wars.stats.wins',
 	],
+	MODES: {
+		ARCADE: {
+			"DAYONE": 'Blocking Dead',
```
Comment (amosyu2000) [inline src/constants/hypixel/Hypixel.js:265]:
```suggestion
			DAYONE: 'Blocking Dead',
```
Please remove quotation marks from the keys in this file.

File: src/constants/hypixel/Hypixel.js [hunk 2]
```diff
@@ -260,4 +260,221 @@ export const HYPIXEL = {
...
+		},
+		BATTLEGROUND: {
+			"mini_ctf": 'Capture the Flag',
+			"domination": 'Domination',
+			"team_deathmatch": 'Team Deathmatch',
+		},
+		BEDWARS: {
+			'BEDWARS_EIGHT_ONE': 'Solo',
+			'BEDWARS_EIGHT_TWO': 'Doubles',
+			"BEDWARS_EIGHT_TWO_TOURNEY": "Doubles (Tournament)",
```
Comment (amosyu2000) [inline src/constants/hypixel/Hypixel.js:302]:
```suggestion
			"BEDWARS_EIGHT_TWO_TOURNEY": 'Doubles (Tournament)',
```
Please only use single-quotes (`'`) for values in this file.

File: src/constants/hypixel/Hypixel.js (hunk 1 above)
Comment (Arjun99ab) [reply src/constants/hypixel/Hypixel.js:265]: Got it. Looking back, I think I kept keys without quotations, however for a key like 1v1 or 2v2, it errors.
image
What should I do about keys that start with numbers?

File: src/constants/hypixel/Hypixel.js (hunk 2 above)
Comment (Arjun99ab) [reply src/constants/hypixel/Hypixel.js:302]: Got it.

File: src/constants/hypixel/Hypixel.js (hunk 1 above)
Comment (amosyu2000) [reply src/constants/hypixel/Hypixel.js:265]: Ahh, in that case, you can single quote them → `'1v1'`

File: src/constants/hypixel/Hypixel.js [hunk 3]
```diff
@@ -260,4 +260,221 @@ export const HYPIXEL = {
...
+			DUELS_CAPTURE_THREES: 'Bridge CTF 3v3',
+			DUELS_DUEL_ARENA: 'Duel Arena',
+		},
+		GINGERBREAD: {},
+		HOUSING: {},
+		LEGACY: {},
+		MAIN: {},
+		MCGO: {
+			'normal': 'Defusal',
+			'gungame': 'Gun Game',
```
Comment (amosyu2000) [inline src/constants/hypixel/Hypixel.js:376]: These might have slipped through

**Review by amosyu2000 — APPROVED** [review]
lgtm

**Pull request merged**